OS X: sed complains about invalid code

Fails with below error,

    11:56:04 $make
    cp ./install_host_app.sh ../bin/testing/install_host_app.sh
    sed -i "s/_VERSIONHOLDER_/testing/g" ../bin/testing/install_host_app.sh
    sed: 1: "../bin/testing/install_ ...": invalid command code .
    make: *** [../bin/testing/install_host_app.sh] Error 1

--- a/src/Makefile
+++ b/src/Makefile
@@ -25,8 +25,7 @@ all: $(HOST_TARGETS)
 	@touch $@
 
 $(HOST_TARGETS): $(TARGET_DIR)/%: $(SRC_DIR)/% $(TARGET_DIR)/.d
-	cp $(SRC_DIR)/$* $@
-	sed -i "s/_VERSIONHOLDER_/$(VERSION)/g" $@
+	sed -e "s/_VERSIONHOLDER_/$(VERSION)/g" < $(SRC_DIR)/$* > $@
 
 clean:
 	rm -rf $(TARGET_DIR)

